<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>management</class>
 <widget class="QWidget" name="management">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="sonListWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>651</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 R</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(202, 202, 202);</string>
   </property>
   <item>
    <property name="text">
     <string>所有</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>图书</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>衣服</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>食品</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="discount_lb">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 B</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>折扣：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="discount_line">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 220, 220);</string>
   </property>
  </widget>
  <widget class="QLabel" name="discount_lb2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 B</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="discount_bt">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 M</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(95, 147, 29);</string>
   </property>
   <property name="text">
    <string>打折</string>
   </property>
  </widget>
  <widget class="QLabel" name="upSuccesslb">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>71</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打折成功！</string>
   </property>
  </widget>
  <widget class="QLabel" name="upFail_lb">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>阿里巴巴普惠体 L</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>打折失败！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetbt">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/static/refresh.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
